Makefile

```diff
@@ -1,11 +1,29 @@
 .PHONY: run
 .ONESHELL:
 run:
-	docker-compose down -v
 	export YAML_CONFIG_FILE_VALUE_AU="$$(cat contract-event-listener-au.yml)"
 	export YAML_CONFIG_FILE_VALUE_GB="$$(cat contract-event-listener-gb.yml)"
+	cat docker-compose.base.yml docker-compose.system.yml > docker-compose.yml
+	docker-compose down -v
+	docker-compose up --build --remove-orphans --renew-anon-volumes
+
+
+.PHONY: run-contract-event-listener
+.ONESHELL:
+run-contract-event-listener:
+	cat docker-compose.base.yml docker-compose.contract-event-listener.yml > docker-compose.yml
```

Comment:
it could be simpler just to list the docker-compose files from the docker-compose commmand, like `docker-compose -f a.yml -f b.yml ...` but this solution is also fine

Reply:
It won't work because of yaml anchors, docker-compose for some reason doesn't support them yet it multifile scenario.